src/DistributionBundle/Command/CreateInitialBackendUserCommand.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+/*
+ * This file is part of the Chameleon System (https://www.chameleonsystem.com).
+ *
+ * (c) ESONO AG (https://www.esono.de)
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace ChameleonSystem\DistributionBundle\Command;
+
+use ChameleonSystem\CoreBundle\Service\LanguageServiceInterface;
+use ChameleonSystem\DistributionBundle\Bootstrap\InitialBackendUserCreator;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+/**
+ * Console command for creating backend users.
+ */
+class CreateInitialBackendUserCommand extends Command
+{
+    /**
+     * @var LanguageServiceInterface
+     */
+    private $languageService;
+    /**
+     * @var InitialBackendUserCreator
+     */
+    private $initialBackendUserCreator;
+
+    public function __construct(InitialBackendUserCreator $initialBackendUserCreator)
+    {
+        parent::__construct('chameleon_system:bootstrap:create_initial_backend_user');
+        $this->initialBackendUserCreator = $initialBackendUserCreator;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function configure()
+    {
+        $this
+            ->setDefinition(array())
+            ->setDescription('Creates a backend user.')
+            ->setHelp(<<<EOF
+The <info>%command.name%</info> creates an initial Chameleon backend user, either by accepting arguments from the environment
```

Comment:
I'd remove the `the` or add a `command` after the name.

Reply:
done